### PR TITLE
Updated monthly statistics

### PR DESCRIPTION
**Pull request by zapbot**
Signed-off-by: zapbot <12745184+zapbot@users.noreply.github.com>

### CODE REVIEW

File: site/data/charts/headline.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-month: March 2024
-zap_runs: 5,894,883
-zap_ascans: 948,187
-zap_alerts: 1,205,768,734
-zap_attacks: 3,295,817,815
+month: April 2024
+zap_runs: 5,464,737
+zap_ascans: 60,231,784
```
Comment (kingthorin) [inline site/data/charts/headline.yaml:3]: Ummm up 60x ??

File: site/data/charts/docker.json [hunk 2]
```diff
@@ -1,89 +1,90 @@
 {
   "title": "Docker Pulls",
   "description": "Docker pulls since the ZAP Docker images were published.",
-  "columns": ["Version" , "live", "stable", "weekly", "bare", "zap-bare", "zap-nightly", "zap-stable", "zap-weekly"],
+  "columns": ["Version" , "live", "stable", "weekly", "zap2docker-bare", "zap2docker-live", "zap2docker-stable", "zap2docker-weekly", "zap-bare", "zap-nightly", "zap-stable", "zap-weekly"],
```
Comment (thc202) [inline site/data/charts/docker.json:4]: Should these ones be back?

Comment (psiinon) [reply site/data/charts/docker.json:4]: Probably yes - I'll look into it, and the jump in ascans...

Comment (psiinon) [reply site/data/charts/docker.json:4]: This should fix that problem: https://github.com/zapbot/zap-mgmt-scripts/pull/188
Looks good locally but I still need to sanity check the ascans..

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**